scipy/stats/tests/test_kdeoth.py

```diff
@@ -85,6 +108,26 @@ def test_kde_2d():
     # get kde for original sample
     gkde = stats.gaussian_kde(xn)
 
+    # evaluate vs multivariate normal, using the KDE definition
+    xx = np.asarray([[1, 2], [3, 4], [5, 6]])
+    arg = xx[:, None, :] - gkde.dataset.T
+    pdf = stats.multivariate_normal.pdf
+    assert_allclose(
+        gkde(xx.T),
```

Comment:
For instance,
```
gkde(xx.T)
[5.35496592e-02 3.93564062e-04 6.68839968e-14]
```
so with `atol=1e-14`, the test with `[5, 6]` is not doing much.